Source/ThirdParty/ANGLE/Configurations/BaseTarget.xcconfig

```diff
@@ -29,7 +29,7 @@ HEADER_SEARCH_PATHS = ${DERIVED_FILES_DIR} $(ANGLE_HEADER_PATH_PREFIX) $(ANGLE_H
 
 USE_HEADERMAP = NO;
 
-GCC_PREPROCESSOR_DEFINITIONS = $(DEBUG_DEFINES) $(WK_OS_UNFAIR_LOCK_INLINE_CFLAGS) LIBGLESV2_IMPLEMENTATION LIBANGLE_IMPLEMENTATION ANGLE_CAPTURE_ENABLED=0 ANGLE_ENABLE_METAL ANGLE_METAL_XCODE_BUILDS_SHADERS ANGLE_ENABLE_METAL_OWNERSHIP_IDENTITY $(GCC_PREPROCESSOR_DEFINITIONS_$(WK_PLATFORM_NAME)) $(GCC_PREPROCESSOR_DEFINITIONS_LIBFUZZER_$(ENABLE_LIBFUZZER));
+GCC_PREPROCESSOR_DEFINITIONS = $(DEBUG_DEFINES) $(WK_OS_UNFAIR_LOCK_INLINE_CFLAGS) LIBGLESV2_IMPLEMENTATION LIBANGLE_IMPLEMENTATION ANGLE_CAPTURE_ENABLED=0 ANGLE_ENABLE_METAL ANGLE_METAL_XCODE_BUILDS_SHADERS ANGLE_ENABLE_METAL_OWNERSHIP_IDENTITY ANGLE_DISABLE_POOL_ALLOC $(GCC_PREPROCESSOR_DEFINITIONS_$(WK_PLATFORM_NAME)) $(GCC_PREPROCESSOR_DEFINITIONS_LIBFUZZER_$(ENABLE_LIBFUZZER));
```

Comment:
How about adding also add `ANGLE_DISABLE_POOL_ALLOC` to the `ANGLE_DEFINITIONS` variable in the `Source/ThirdParty/ANGLE/CMakeLists.txt` for the other ports?